src/HasStates.php

```diff
@@ -181,4 +181,57 @@ private static function getStateConfig(): array
 
         return static::$stateFields ?? [];
     }
+
+    /**
+     * Get the registered states for the model.
+     *
+     * @return \Illuminate\Support\Collection
+     */
+    public static function getStates(): \Illuminate\Support\Collection
+    {
+        return collect(static::getStateConfig())
+            ->map(function ($state) {
+                return $state->stateClass::all()
+                    ->map(function ($state) {
+                        return new $state(new static);
```

Comment:
Other functionality provided by this package returns the morphClass instead of state instances. I think it'd be good to keep this consistent.